packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -81,9 +87,12 @@ const Accordion = ({ children }: AccordionProps) => {
   const elementId = useId();
 
   return (
-    <div className="Accordion">
+    <div className="Accordion" {...props}>
       <ExpandCollapsePanel
         id={panelElement.props.id || `${elementId}-panel`}
+        open={open}
+        onToggle={onToggle}
+        animationTiming={animationTiming}
```

Comment:
We are already spreading `{...panelElement.props}`. Why not have these props be part of the panel's props (`AccordionContentProps`)?

Reply:
I realize that the `AccordionProps` extend from `ExpandCollapsePanel`, but since that wasn't actually implemented, I don't think it would be a breaking change to make `AccordionContentProps` extend from `ExpandCollapsePanel` instead

Reply:
Because they're 2 different elements. The resulting HTML looks something like this:

```jsx
<div class="Accordion">
  <button>...</button>
  <div class="ExpandCollapse__Panel">...</div>
</div>
```

The accordion props should apply to the `.Accordion` element. The panel props should apply to the `.ExpandCollapse__Panel` element. They need to be able to set html attributes independently of one another.

Reply:
That's an pre-existing issue, would you mind raising a new issue for that?